Outline updates (#200)
greenelab/covid19-review: content/02.introduction.md

On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf?sfvrsn=99251b2b_4].
At this time, over 92,000 deaths had been reported due to COVID-19 worldwide.
New information is emerging daily about the impact of the disease around the world.

> COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).

Reviewer: At the end will we stick to only one abbreviation of the current virus? I remember using COVID-19, others using SARS-CoV-2 etc.  Although I guess COVID-19 refers to the infection itself rather than the virus for which SAR-CoV-2 denotes, so my point is probably invalid.

Reviewer: I think you're right on the SARS-CoV-2 being the virus and COVID being the disease. Your proposals sound good :)

> COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).

Reviewer 2: ```suggestion
COVID-19 is a disease caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).
```

> SARS-CoV-2 belongs to coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.

Reviewer 3: ```suggestion
SARS-CoV-2 belongs to the family of coronaviruses, which are RNA viruses known to cause respiratory and intestinal infections in humans and other species.
```

Reviewer 2: I think i would say something like "Coronavirus is a genus of Coronaviridae that is known to ...." The current sentence implies to me, at least, that coronaviruses have this attribute of causing respiratory and intestinal infections as opposed to Toroviruses, which I don't think is true

> Infectious diseases of global concern have previously been associated with coronaviruses, including Severe Acute Respiratory Syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9]; however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).

Reviewer: ```suggestion
Infectious diseases of global concern have previously been associated with coronaviruses, including severe acute respiratory syndrome (SARS-CoV) and Middle East respiratory syndrome (MERS-CoV) [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9]; however, neither of these reached pandemic status, owing to proper containment procedures (SARS) or intrinsic limitations in virus transmission (MERS).
```

Reviewer 3: I'm not sure what to do with caps here. Suggest either caps all of SARS and MERS or agree with @RLordan's change.

Additionally, there are four endemic human coronaviruses that rarely progress beyond the mild symptoms associated with the common cold [@doi:10.1038/s41579-018-0118-9].

> The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].

Reviewer: maybe identify the disease rather than diagnose? up to you :)

Author: This should be addressed in suggestions! Thanks @RLordan

> The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].

Reviewer: Yes I think this is correct. Thanks @rando2 

Good for them, I wish I was being as useful!!!

> The precise identity of SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].

Author: @RLordan I feel the same way but you are being extremely helpful here!

As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### 2.2 Coronaviruses

#### 2.2.a Classification
Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses [@isbn:0781760607]. 

> Members of this family include the species Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV) [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].

Reviewer: ```suggestion
Members of this family include the species Middle East respiratory syndrome-related coronavirus (MERS-CoV) and severe acute respiratory syndrome coronavirus (SARS-CoV) [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
```

Reviewer 3: I don't think you have to redefine this acronym. It's earlier in the intro :)

Reviewer 2: Agreed..didn't realize that when I wrote it initially!

> Members of this family include the species MERS-CoV and SARS-CoV [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].

Reviewer 4: this is outdated per my pr yesterday

Author: I'll fix it on merge :)

The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].  

#### 2.2.d Transmission
In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes). 
While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it has been suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies [@doi:10.3390/ijerph15122757].

### 2.3 SARS-CoV-2: Origin, Phylogenetics, and Evolution

#### 2.3.a Origin of SARS-CoV-2

> The origin of the SARS-CoV-2 virus is not yet fully understood.

Reviewer 3: Does this mean the species that it leapt from to humans? I guess at this point I view it as safe to say the origin is zoonotic transmission but the previous host or hosts remains unknown.

Author: How about:
```suggestion
The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
```
(I think Ronnie wrote this, but @RLordan added related text elsewhere if I recall correctly, so he might also have an opinion!)

Reviewer 2: I envisioned this section to talk about the phylogenetic analyses that have been done to examine the intermediate host. I think there are some predictions but nothing conclusive from what I can see.

Author: @ajlee21 does this way look OK to you? Or did you mean something else? Also, sorry that I forgot you were the one who wrote this, I should have checked the blame!

Therefore, the fields of virology, immunology, and molecular biology are fundamental to characterizing SARS-CoV-2 and COVID-19.
These topics can be approached using a range of techniques, including characterization of the host response from the cellular to systems level.
Contextualizing SARS-CoV-2 in relation to other viruses that infect humans and other animals can further serve to elucidate the reaction of the human host to viral exposure.
This information, when combined with an understanding of the biology of pharmaceutical and medical interventions, can guide new approaches to treatment.

#### Public Health

One necessary component of determining how to manage the outbreak is to understand epidemiological factors related to the transmission of the SARS-CoV-2 virus. 
These can include characteristics such as when infected individuals are contagious, how the virus is transmitted between individuals, the range of symptoms associated with infection and/or contagiousness in different individuals, and how rapidly the virus propogates between individuals, etc.

> The development of diagnostic tools is critical to this goal.

Reviewer 4: you mean better or more accurate diagnostic tools? because tools exist already

Author: I think we are aiming to review all of the diagnostic tools developed for SARS-CoV-2 (even if some of them were developed in January) so I think this works -- but let me know if I'm missing something!

> Accurate diagnoses on a large scale is necessary for collected the data needed to develop epidemiological models.

Reviewer 3: ```suggestion
Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
```

##### 2.5.c.ii Therapeutics

The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus' virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 are thus critical to identifying candidate therapeutics.

### 2.6 Summary
In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.

> We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review.

Reviewer: Maybe make a comment along the line of " but have been critically appraised". It might not be necessary though if you have mentioned this in the methodology sections.

Author: Like this?
```suggestion
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically appraised by the scientists involved in this effort.
```

Reviewer: Perfect, exactly
